MIT_GEOS5PlugMod/CMakeLists.txt

```
@@ -0,0 +1,118 @@
+esma_set_this ()
+
+set (SRCS
+  MIT_GEOS5PlugMod.F90
+  )
+
+if (NOT DEFINED MIT_CONFIG_ID)
+   message("You need to specify MIT_CONFIG_ID")
```

Do you want to make this a fail message?
```suggestion
   message(FATAL_ERROR "You need to specify MIT_CONFIG_ID")
```

---

Thanks @tclune , ✅ https://github.com/GEOS-ESM/GEOS_OceanGridComp/commit/5b21b29979951507cae695655a6000c1c3214930